doc/api/n-api.md

```diff
@@ -177,6 +177,19 @@ developers have run into limitations in node-gyp.
 CMake.js is a good choice for projects that already use CMake or for
 developers affected by limitations in node-gyp.
 
+#### vcpkg
+
+For vcpkg users there are node-api and node-addon-api ports that can be
+installed by the following commands.
+
+```bash
+vcpkg install node-api
```

Comment:
I didn't find these packages on https://vcpkg.io/en/packages.html and I'm not aware of an existing release process that publishes these packages.

@mhdawson please correct me if I'm missing it.

Reply:
Looks like these are referencing the ports added by:
- https://github.com/microsoft/vcpkg/pull/26768
- https://github.com/microsoft/vcpkg/pull/28427

Reply:
> Looks like these are referencing the ports added by:

Yeah, that's true. I'm not sure what revision of vcpkg does the site use, but those packages are available for users.

Unlike many package managers vcpkg consists of build recipes instead of storing files in the cloud. Those build recipes are being maintained by the community and Microsoft.

The usual workflow is to have vcpkg repo cloned somewhere on your machine. So, once you cloned latest vcpkg or pulled latest changes, you'd be able to run `vcpkg install node-api` and `vcpkg install node-addon-api`.

After running those commands you can see a brief explanation on how to use node-api/node-addon-api with your existing CMake project (and some CMake lines to copy-paste).